GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -572,7 +578,8 @@ private synchronized int writeInstanceData(Sheet sheet, final int startRow,
             InstanceData instanceData, boolean generateSummary,
             List<String> questionIdList, List<String> unsummarizable,
             Map<String, String> nameToIdMap, Map<String, String> collapseIdMap,
-            SummaryModel model, boolean useQuestionId)
+            SummaryModel model, boolean useQuestionId, 
+            String displayName)
```

Comment:
Rather than add an extra parameter here, I would be in favour of having `displayNamePartList` as a global since it does not change over the entire survey generation..  Otherwise it feels strange on line 554 to be passing in the `instanceData` twice. i.e. as one of parameters and in addition nested in the `constructDisplayName` call